### ISSUE TITLE
Convergence always does not update default timeout

### ISSUE THREAD

**Issue by wwilsman** [thefrontside/bigtest#66]
By default, using `.always()` sets it's timeout to be one-tenth of the convergence timeout. When the convergence timeout is updated using `.timeout()`, the old one tenth timeout is still used for that `.always` convergence in the stack.

```
new Convergence()
  .timeout(5000).always(() => true) // 50ms default timeout
  .timeout(4000).always(() => true) // 40ms default timeout
  .timeout(3000).always(() => true) // 30ms default timeout
  .do(() => {}) // ensures the previous always uses it's own timeout
  .timeout(100).run() // takes at least 120ms, fails to be within 100ms
```

When no timeout is provided to the `.always()` method, we should calculate the default one-tenth timeout when `.run()` is called. In this example, the minimum `20ms` timeout would be used and the whole convergence would take `60ms`.

**Comment by Robdel12** [comment]
This seems sort of like a bug, no?

**Comment by wwilsman** [comment]
This was fixed in the run refactor that went with #67
